coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/migration/Migration.scala

```diff
@@ -26,7 +27,7 @@ object Migration {
                 operation: MigrationOperation = MigrationOperation.Migrate,
                 numChanges: Int = 1,
                 changeLogPath: String = MigrationScriptPath) {
-
+    LogFactory.getLogger().setLogLevel("warning")
```

Comment:
I like to cut down logging.  But I find it helpful to have INFO in migration.  Feel better w/o going to check the database if and which migration is done.